이건우/week9/leetcode413.py

```diff
@@ -0,0 +1,7 @@
+class Solution(object):
+    def fizzBuzz(self, n):
+        """
+        :type n: int
+        :rtype: List[str]
+        """
+        return ["FizzBuzz" if (i%15 == 0) else "Fizz" if (i%3 == 0) else "Buzz" if (i%5 == 0) else str(i) for i in range(1,int(n) + 1)]
```

Comment:
n이 이미 int인데 int로 캐스팅할 필요가 있을까요?
```suggestion
        return ["FizzBuzz" if (i%15 == 0) else "Fizz" if (i%3 == 0) else "Buzz" if (i%5 == 0) else str(i) for i in range(1, n + 1)]
```
그리고 삼항연산자가 여러번에 거쳐서 직렬로 연결되어 있어서 가독성이 떨어지는 것 같아요 :(

Reply:
엇 그러네용 ..